src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/AbstractProjectConfigurationProperties.cs

```diff
@@ -218,5 +218,6 @@ public string DebugInfo
         public bool CodeAnalysisIgnoreBuiltInRules { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
         public bool CodeAnalysisFailOnMissingRules { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
         public bool Prefer32Bit { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+        public bool PreferNativeArm64 { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
```

Comment:
You don't need to add this. This class serves as the base implementation of the `ProjectConfigurationProperties3` interface; it doesn't need any functionality that isn't exposed by that interface.

Reply:
removed!